language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -16,6 +16,19 @@ use move_binary_format::{
     file_format::{CompiledModule, CompiledScript},
 };
 
+#[derive(Debug, Clone)]
+pub struct VerifierConfig {
+    pub max_loop_depth: usize,
```

Comment:
Might be a bit cleaner to just make this an `Option`?

And thoughts on sticking in the config for the script verifier? Making the default `no_additional_script_signature_checks`?

Reply:
You mean having something like `fn verify_script(config: &VerifierConfig, ..)` and `fn verify_script_with_default_config(..)`?

Reply:
NVM I think I misunderstood. 

> And thoughts on sticking in the config for the script verifier? Making the default 
I get you mean. Sounds like a great idea. We can do that in a separate PR